src/styles/card.scss

```diff
@@ -0,0 +1,115 @@
+.card-container {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  gap: 48px 46px;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  grid-template-rows: repeat(2, 406px);
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    --column-count: 2;
+  }
+
+  @media (min-width: 768px) {
+    --column-count: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --column-count: 4;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 23px;
+  }
+
+  &__description {
+    padding: 0 16px;
+    margin-top: 5px;
+  }
+
+  &__description p:first-child {
+    line-height: 18px;
+    margin-bottom: 0;
+  }
+
+  &__description p:last-child {
+    font-size: 10px;
+    font-weight: 400;
+    color: $additional-color;
+    margin-top: 5px;
+    margin-bottom: 9px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 16px;
+    margin-top: 9px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    padding: 0 16px;
+    margin: 27px 0 19px;
+
+    & div:first-child {
+      color: $additional-color;
+      font-size: 12px;
+      line-height: 11px;
+      font-weight: 400;
+      margin: 0;
+    }
+
+    & div:last-child {
+      font-size: 16px;
+      line-height: 12px;
+      font-weight: 700;
+      margin: 0;
+      vertical-align: baseline;
+    }
+  }
+
+  &__button {
+    display: block;
+    text-align: center;
+    padding: 12px;
+    margin: 0 16px 16px;
+    background-color: $blue-color;
+    color: #fff;
+    border-radius: 5px;
+    text-decoration: none;
+    cursor: pointer;
+
+    font-size: 14px;
+    font-weight: 700;
+  }
+  &__button:hover {
+    border: 1px solid $blue-color;
```

Comment:
The variable `$blue-color` is used here but is not defined in this file. Ensure that it is defined in a global SCSS file or imported correctly to avoid compilation errors.